template-reactless/my_component/example.py

```diff
@@ -0,0 +1,28 @@
+import streamlit as st
+from my_component import my_component
+
+# Add some test code to play with the component while it's in development.
+# During development, we can run this just as we would any other Streamlit
+# app: `$ streamlit run my_component/example.py`
+
+st.subheader("Component with constant args")
+
+# Create an instance of our component with a constant `name` arg, and
+# print its output value.
+num_clicks = my_component("World")
+st.markdown("You've clicked %s times!" % int(num_clicks))
+
+st.markdown("---")
+st.subheader("Component with variable args")
+
+# Create a second instance of our component whose `name` arg will vary
+# based on a text_input widget.
+#
+# We use the special "key" argument to assign a fixed identity to this
+# component instance. By default, when a component's arguments change,
+# it is considered a new instance and will be re-mounted on the frontend
+# and lose its current state. In this case, we want to vary the component's
+# "name" argument without having it get recreated.
+name_input = st.text_input("Enter a name", value="Streamlit")
+num_clicks = my_component(name_input, key="foo")
+st.markdown("You've clicked %s times!" % int(num_clicks))
```

Comment:
Do we need to update the component guide now?

Reply:
<img width="965" alt="Screenshot 2023-08-22 at 11 18 00" src="https://github.com/streamlit/component-template/assets/78743291/4b784057-9f7d-48cf-98ba-bf581df0fd7e">